src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -231,25 +254,23 @@ const NavHeader = (props) => {
           />
         </View>
       )}
-    </View>
+    </Animated.View>
   );
 };
 
 NavHeader.propTypes = {
-  initialData: PropTypes.object,
   onShareCommunity: PropTypes.func,
   isHeaderHide: PropTypes.bool,
   isFollow: PropTypes.bool,
   hideSeeMember: PropTypes.bool,
   animatedHeight: PropTypes.number,
   opacityHeaderAnimation: PropTypes.number,
-  opacityImage: PropTypes.number,
   getTopicDetail: PropTypes.func,
   setMemberCount: PropTypes.func,
   memberCount: PropTypes.number,
   topicDetail: PropTypes.object,
   setIsFollow: PropTypes.func,
-  isLoading: PropTypes.bool,
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `initialData`, `isLoading`, and `opacityImage` from `NavHeader.propTypes` might cause issues if they were used in the component. Please ensure that these props are not required elsewhere in the component or by any components that use `NavHeader`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->